src/diagnostics/algolia-API-timing.js

```diff
@@ -15,8 +15,11 @@ function algoliaAPITiming(cb) {
   }
 
   var async = require('async');
+  var querystring = require('querystring');
 
-  var appId = 'latency';
+  var q = querystring.parse(document.location.search.slice(1));
+
+  var appId = (q.applicationId || 'latency').toLowerCase();
```

Comment:
I think this variable is given in another way than the query string too, is this similar to another part of the code?

Reply:
The only place where I found a reference to `applicationId` or `appId` is [here](https://github.com/algolia/diag/blob/master/src/diagnostics/empty-search.js#L13) (the code that does the empty search). So I did the same.